app/views/questions/welcome/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :page_title, t("views.questions.welcome.title") %>
```

Comment:
the view (and associated i18n string) doesn't need to exist anymore if the controller is just a redirect